### PR TITLE
[Compiler] fix object creation

### PR DESCRIPTION
**Pull request by Jim8y**
This pr fix the problem that we can not create Map or List object with initializer. 

The reason of this is that Map and List has no explicitly defined Property.

### CODE REVIEW

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs [hunk 1]
```diff
@@ -13,15 +17,44 @@ public static void testProperty2()
...
+        {
+            1, 2, 3, 4, 5
+        };
+    }
+
     public class Person
     {
         public string Name { get; set; }
-        public int Age { get; }
+        public int Age { get; set; }
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs:48]: We remove the test without the setter

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs [hunk 2]
```diff
@@ -1,3 +1,7 @@
+
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs:1]:
```suggestion
```

File: tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs (hunk 1 above)
Comment (Jim8y) [reply tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs:47]: i dont get that?

Comment (shargon) [reply tests/Neo.Compiler.CSharp.TestContracts/Contract_PropertyMethod.cs:47]: This ensure that a peoperty without set, initialized in the constructor works

File: tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Property_Method.cs [hunk 3]
```diff
@@ -1,7 +1,10 @@
+using System;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Neo.SmartContract.Testing;
 using Neo.VM.Types;
 using System.Numerics;
+using Akka.Util.Internal;
```
Comment (shargon) [inline tests/Neo.Compiler.CSharp.UnitTests/UnitTest_Property_Method.cs:6]:
```suggestion
```
It's required?

**Review by shargon — APPROVED** [review]

**Pull request merged**